src/ethereum/curve/contracts.ts

```diff
@@ -1699,4781 +6477,7 @@ const contracts = {
-      tokens: ["0x465B0673BbcECA807504FEB129d67576c698c649", "0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490"],
-    },
-    "usdf_usdf+3crv": {
-      pool: "0x6E386F8E746af332d18f5c21F3369835C9c5DB16",
-      lpToken: "0x6E386F8E746af332d18f5c21F3369835C9c5DB16",
-      tokens: ["0x92FC8E6EfdF389e2527d14393b15F543f9a03420"],
-    },
-    "3crv_usdf+3crv": {
-      pool: "0x6E386F8E746af332d18f5c21F3369835C9c5DB16",
-      lpToken: "0x6E386F8E746af332d18f5c21F3369835C9c5DB16",
-      tokens: ["0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490"],
-    },
-    "usdf+3crv": {
-      pool: "0x6E386F8E746af332d18f5c21F3369835C9c5DB16",
-      lpToken: "0x6E386F8E746af332d18f5c21F3369835C9c5DB16",
-      tokens: ["0x92FC8E6EfdF389e2527d14393b15F543f9a03420", "0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490"],
-    },
-    "2-nusd_nusd+3crv": {
-      pool: "0x57EB6fbE61216d9f8c7c09Ad1EE5a3023747244b",
-      lpToken: "0x57EB6fbE61216d9f8c7c09Ad1EE5a3023747244b",
-      tokens: ["0x1BEf2e5DE862034Fb0ed456DF59d29Ecadc9934C"],
-    },
-    "2-3crv_nusd+3crv": {
-      pool: "0x57EB6fbE61216d9f8c7c09Ad1EE5a3023747244b",
-      lpToken: "0x57EB6fbE61216d9f8c7c09Ad1EE5a3023747244b",
-      tokens: ["0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490"],
-    },
-    "2-nusd+3crv": {
-      pool: "0x57EB6fbE61216d9f8c7c09Ad1EE5a3023747244b",
-      lpToken: "0x57EB6fbE61216d9f8c7c09Ad1EE5a3023747244b",
-      tokens: ["0x1BEf2e5DE862034Fb0ed456DF59d29Ecadc9934C", "0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490"],
-    },
-    "ousd_ousd+3crv": {
-      pool: "0x87650D7bbfC3A9F10587d7778206671719d9910D",
-      lpToken: "0x87650D7bbfC3A9F10587d7778206671719d9910D",
-      tokens: ["0x2A8e1E676Ec238d8A992307B495b45B3fEAa5e86"],
-    },
-    "3crv_ousd+3crv": {
-      pool: "0x87650D7bbfC3A9F10587d7778206671719d9910D",
-      lpToken: "0x87650D7bbfC3A9F10587d7778206671719d9910D",
-      tokens: ["0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490"],
-    },
-    "ousd+3crv": {
-      pool: "0x87650D7bbfC3A9F10587d7778206671719d9910D",
-      lpToken: "0x87650D7bbfC3A9F10587d7778206671719d9910D",
-      tokens: ["0x2A8e1E676Ec238d8A992307B495b45B3fEAa5e86", "0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490"],
-    },
-    "dsu_dsu+3crv": {
-      pool: "0x6eC80df362d7042c50D4469bcfbc174C9dd9109a",
-      lpToken: "0x6eC80df362d7042c50D4469bcfbc174C9dd9109a",
-      tokens: ["0x605D26FBd5be761089281d5cec2Ce86eeA667109"],
-    },
-    "3crv_dsu+3crv": {
-      pool: "0x6eC80df362d7042c50D4469bcfbc174C9dd9109a",
-      lpToken: "0x6eC80df362d7042c50D4469bcfbc174C9dd9109a",
-      tokens: ["0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490"],
-    },
-    "dsu+3crv": {
-      pool: "0x6eC80df362d7042c50D4469bcfbc174C9dd9109a",
-      lpToken: "0x6eC80df362d7042c50D4469bcfbc174C9dd9109a",
-      tokens: ["0x605D26FBd5be761089281d5cec2Ce86eeA667109", "0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490"],
-    },
-  },
-  CurveMetapoolGaugePool: {
+  CurveMetapoolGauge: {
```

Comment:
Small Doubt: Don't we need to keep pools under `pools` property here? Ot it is fine if we keep pools directly under `CurveMetapoolGauge` key?

Reply:
I didn't add subkeys because the CurveMetapoolGauge ABI is the same as CurveGauge ABI. Does it make sense?

Reply:
Discussed!